src/backend/DocDownloaderAPI/src/main/resources/application.properties

```diff
@@ -1,4 +1,5 @@
 # Downloader API configuration data 
+server.port=8088
```

Comment:
Let's not change the server port for the API as it will muck up the OpenShift integration during containerization. Typically this is left out (defaults to 8080).